src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -308,3 +310,96 @@ export const answerReviewLabel = (key, formValues) => {
     }
   }
 };
+
+export const determineBoardObj = (formResponses, noDRB) => {
+  if (!formResponses) return null;
```

Comment:
Consider: We use curly braces in all other logic. For consistency, perhaps we should use them every time.